### DISCUSSION TITLE
Hide and Details as one menu item

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#20983] [Ideas]
I see that `renderingProperty`
for `category="details"` and `category="hide" `always use `type="boolean" `if they have to be seen in menu Map rendering: Hide or Detailes.
If someone use e.g. `type="string" `then `renderingProperty` is moved to different menu like `Other` map attributes.
Difference beteween Hide and Detailes for booleans is only that 1st uses checkbox and 2nd uses switch.
My idea is to stop useing Hide.

Generally it depends on rendering file but I noticed that some items are hardcoded.
Hide military area sometimes appears in Hide menu despite the fact that I defind it in Detailes. (I not find the reason yet and it is dificult to reproduce so I will report this 
later) But anyway it is not possible to change `name="Hide military areas" `until I use `attr="hideMilitaryAreas"`

The same with Water, boundaries ...
It is hardcoded.

But the main idea is to use Detailes only for boolean.
Hide seems to be usless.

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/e4169bec-b4c3-45f5-bbf5-3d77cc794f18"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by vshcherb** [comment]
True, though we plan to move out details first to separate sections similar to Routes and then deal only with "hide" actions. So it's in plans

**Comment by mariush444** [comment]
Thank you for answer.

just please noticed that
`name` is hardcoded so **Hide** water in **hide** menu is redundand/doubled.

I changed `attr` to achive my needs.

Additionaly `description` is not shown to a user at all.
